File: src/001/p001.js
function solve() {
  return Array(999)
    .fill(0)
    .map((value, index) => index + 1)
    .filter(value => value % 3 === 0 || value % 5 === 0)
    .reduce((sum, value) => sum + value)
}

const result = solve()
console.log(result)

File: src/001/p001.scala
object Main {
  def solve(): Int = {
    (0 to 999).filter((x) => ((x % 3) == 0 || (x % 5) == 0)).sum
  }

  def main(args: Array[String]): Unit = {
    val result = solve()
    println(result)
  }
}